Products/ATContentTypes/content/topic.py

```diff
@@ -147,7 +147,7 @@ class ATTopic(ATCTFolder):
 
     use_folder_tabs = 0
 
-    implements(IATTopic, IDisabledExport)
+    implements(IATTopic)
```

Comment:
I'd agree that the import can be removed, but I'd err on the side of caution about removing the entire feature, in case people are depending on it. If someone had an issue with this feature for whatever reason then leaving IDisabledExport in would let them turn it off again quickly. It may also (unlikely…) be used in third party code?